CMakeLists.txt

```
@@ -34,7 +34,7 @@ endif()
 
 # Geographiclib installs FindGeographicLib.cmake to this non-standard location
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "/usr/share/cmake/geographiclib/")
-find_package(GeographicLib REQUIRED COMPONENTS STATIC)
+find_package(GeographicLib REQUIRED COMPONENTS SHARED)
```

Is there a way to just remove the explicit `STATIC`/`SHARED` dependency on the line?
Something like just
```suggestion
find_package(GeographicLib REQUIRED)
```